docs/website-v2/docs/howtos/README_package.md

```diff
@@ -13,32 +13,32 @@ gateway VM
 Creating a production release package.
 ---------------------------
 Run `fab test package:git` under lte/gateway/ on the host to create a Magma
-release. This runs the `build-magma.sh` script with the latest commit id with 
-production compiler flags (`RelWithDebInfo`). 
+release. This runs the `build-magma.sh` script with the latest commit id with
+production compiler flags (`RelWithDebInfo`).
 It builds everything, identifies dependencies (assuming they're specified in the
-setup.py properly), and creates a Debian package in the magma-packages/ 
+setup.py properly), and creates a Debian package in the magma-packages/
 directory on dev VM.
 
 If you want to bump the package version, this can be done inside build-magma.sh.
-The version number should be bumped when there is a minor or major feature or 
-bug fix release. 
-If you're just making a minor tweak (e.g., mistake in building the package), 
+The version number should be bumped when there is a minor or major feature or
+bug fix release.
+If you're just making a minor tweak (e.g., mistake in building the package),
 you can also increment the iteration number.
 
 Creating a development package.
 ---------------------------
-To create an AGW package with debug compiler flags (`Debug`), 
-run `fab dev package:git`. 
+To create an AGW package with debug compiler flags (`Debug`),
+run `fab dev package:git`.
 
 Testing a release package before you push it.
 ---------------------------------------------
 You should always do this. In general, try your best not to release broken
 packages.
 
 1. Build the release like you normally would.
-2. Spin up a fresh prod VM or gateway machine and copy the magma_<version>.deb
+2. Spin up a fresh prod VM or gateway machine and copy the magma_&lt;version&gt;.deb
 generated above.
-3. Run `sudo apt-get install gdebi; sudo gdebi magma_<version>.deb' 
+3. Run `sudo apt-get install gdebi; sudo gdebi magma_&lt;version&gt;.deb'
```

Comment:
Docusaurus use MDX (ie React components in markdown) so whenever yu use `<xyz>` you can wrap it in a code block or escape de chars so that MDX understand it's not React/JSX